.github/workflows/nodejs-ci.yml

```diff
@@ -85,6 +91,16 @@ on:
         description: "Command to execute at the end of the container scan"
         type: string
         default: ""
+      send_test_coverage_report_to_pulse:
+        description: "A boolean value to enable sending test coverage report to Pulse"
+        type: boolean
+        required: false
+        default: false
+      test_coverage_report_directory:
```

Comment:
Can we rename it to `dev_test_coverage_report_directory` to keep in sync with other dev_test vars?

Reply:
yes done